projects/templates/projects/project_edit.html

```diff
@@ -0,0 +1,46 @@
+{% extends "index.html" %}
+{% load static %}
+
+{% block title %}
+    <title>Features</title>
+{% endblock %}
+
+
+{% block styling %}
+    <!-- Any styling needed for this template would go here -->
+    <link rel="stylesheet" href="{% static 'CSS/test.css' %}">
+{% endblock %}
+
+
+<!-- Main content of template goes here -->
+{% block content %}
+    <div >
+
+        <li>{{ project_data.id }}</li>
+        <li>{{ project_data.name }}</li>
+        <li>{{ project_data.description }}</li>
+        <li>{{ project_data.due_date }}</li>
+        {% if project_data.status == 0 %}
+            <li>NOT STARTED</li>
+        {% elif project_data.status == 10 %}
+            <li>IN PROGRESS</li>
+        {% elif project_data.status == 20 %}
+            <li>UNDER REVIEW</li>
+        {% elif project_data.status == 30 %}
+            <li>COMPLETED</li>
+        {% else %}
+            <li>STALLED</li>
+        {% endif %}
+        <form id="login-form" method="post" action=""> {% csrf_token %}
+        <span class="form-button-container">
+            <input class="form-button submit" type="submit" value="Complete">
```

Comment:
I think it would make more sense for Status to be a dropdown than a button, but this is probably fine for the purpose of the user story.

Reply:
I have to come back to the page to implement the edit buttons anyway. I'll remove it then.